webapp/QAFs.md

```diff
@@ -0,0 +1,344 @@
+Quality Assured Features (QAFs)
+===============================
+
+Contents:
+- About this software
+- Compatibility
+- Features
+
+____________________________________________
+
+About this software
+-------------------
+
+- Name: Panoptes
+- Type: web application
+- Release: v2.0.beta4
+
+
+Compatibility
+-------------
+
+###Compatible server operating systems (and newer):
```

Comment:
MIght want to take the "newer" off here - as we have had cases where newer OS doesn't work till we fix it.